low_cost_robot_arm/low_cost_robot_arm.xml

```diff
@@ -0,0 +1,136 @@
+<mujoco model="low_cost_robot_arm">
+  <compiler angle="radian" meshdir="./assets/"/>
+
+  <option cone="elliptic" impratio="10"/>
```

Comment:
Please switch to integrator="implicitfast" when using explicit position actuators (aka kp and kv). Also remove joint damping.